bson/json_util.py

```diff
@@ -372,6 +372,7 @@ def with_options(self, **kwargs: Any) -> "JSONOptions":
             >>> json_options.tz_aware
             False
 
+
```

Comment:
Here and above seem to be accidental changes.

Reply:
I needed them to avoid sphinx warnings about indentation.

Reply:
Removed now that we're not using `sphinx-autodoc-typehints`.